i6_models/primitives/feature_extraction.py

```diff
@@ -22,6 +22,9 @@ class LogMelFeatureExtractionV1Config(ModelConfiguration):
         min_amp: minimum amplitude for safe log
         num_filters: number of mel windows
         center: centered STFT with automatic padding
+        periodic: whether the window is assumed to be periodic
+        mel_options: extra options for mel filters
+        rasr_compatible: apply FFT to make features compatible to RASR's, otherwise (default) apply STFT
```

Comment:
instead of making this a bool, I would suggest this should be an enum:
```
class SpectrumType(Enum):
    PYTORCH_STFT=1
    RASR_COMPATIBLE=2
```

Reply:
Why `PYTORCH_STFT`? This is not at all specific about PyTorch?

Either we describe here better what on the actual difference type, spectrum type, then this is here STFT vs FFT (and not about PyTorch vs RASR or so), and mention just in the docs that RASR uses FFT.

Or we describe here on common known libraries which behave this way. But then this is more like librosa vs RASR.

Reply:
The function called below is torch.stft, which is based on librosa stft, but not a drop-in replacement according to the documentation: https://pytorch.org/docs/stable/generated/torch.stft.html . So PYTORCH_STFT is justified in my opinion. The other option could be named RASR_COMPATIBLE_RFFTN if you prefer that.

Reply:
All Torch functions start with `torch.`.

Reply:
Yes, but the STFT implementation seems to be pytorch specific and not quite compatible with what other libraries do (librosa in this case). That's why I think it's OK to highlight that it's pytorch specific.

Reply:
I actually investigated the exact differences on STFT a while ago, between TensorFlow, PyTorch, SciPy and librosa. See for example https://github.com/librosa/librosa/issues/1701 or https://github.com/pytorch/pytorch/issues/100177. They all have slightly different behavior w.r.t. padding. But you can always use any of those STFT functions and perfectly reproduce the behavior of any other STFT function, because they ultimately do the same thing.

Reply:
OK, so what enum values would you prefer to have?

Reply:
I don't really remember anymore, didn't we implement this in a way that we actually exactly get the librosa behavior? In that case, I would maybe use:
```python
class SpectrumType(Enum):
    LIBROSA_STFT=1
    RASR_FFT=2
```
Or is this independent and just our own new implementation? In that case, I would just use:
```python
class SpectrumType(Enum):
    STFT=1
    FFT=2
```
And if we want to have other aspects configurable about it (like type of windowing, e.g. Hanning or sth else, or the kind of padding, etc), I think that would be separate arguments.

Reply:
How about STFT and RFTTN (since these are the 2 pytorch functions we call)?

Reply:
I don't like FFT alone, because that's yet another function in pytorch which will give you the the full FFT output, whereas RFFT will only give you the non-redundant output from the Fourier transform.